ens/utils.py

```diff
@@ -200,7 +185,7 @@ def dot_eth_namehash(name):
     :rtype: bytes
     :raises InvalidName: if ``name`` has invalid syntax
     '''
-    expanded_name = dot_eth_name(name)
+    expanded_name = normalize_name(name)
```

Comment:
The name of `dot_eth_namehash` should also probably change (since it doesn't append `dot_eth` anymore). Maybe `raw_name_to_hash`?

The other similar method, `name_to_hash`, could be changed to `normal_name_to_hash` to help distinguish.